test/fixtures/bad_code.rb

```diff
@@ -0,0 +1,32 @@
+#
+# Note: This file is from devise. It was encountered in the wild and was
```

Comment:
Can we distill down the fixture file to a bare minimum enough to reproduce the erb failure? Then we can get rid of the explanation comment because the code wouldn't come from devise anymore.

Also, can this file be called `erb_template.rb` instead of `bad_code.rb` so it's a little bit more descriptive?